core/server/master/src/test/java/alluxio/master/journal/JournalContextTest.java

```diff
@@ -174,6 +174,22 @@ public void pauseBlocksJournalContext() throws Exception {
     }
   }
 
+  @Test
```

Comment:
nit: To link test case to the real issue for more context in the future.
```suggestion
  // See https://github.com/Alluxio/alluxio/issues/13904
  @Test
```